flamechart/package.json

```diff
@@ -1,6 +1,14 @@
 {
   "name": "@perses-dev/flame-chart",
```

Comment:
should we change the name also to suffix it by `-plugin` ?

Reply:
and I think that's the same for the pyroscope plugin it is not suffix by `-plugin`

Reply:
yes, good catch. This will create new npm packages but I don't think that is an issue

Reply:
yeah we didn't say anything about these new plugins, so we can still break them hehe